server/src/game/Room.ts

```
@@ -391,15 +452,50 @@ class MultiplayerRoom extends Room {
   }
 
   startPlay() {
+    this.gameActionRecord.initialize();
+
     for (let member of this.memberConnectionIDs) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'member' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)